sdk/samples/iot/README.md

```diff
@@ -603,243 +604,62 @@ This section provides an overview of the different samples available to run and
   </p>
   </details>
 
-### IoT Hub Plug and Play Sample
-
-- *Executable:* `paho_iot_hub_pnp_sample`
+### IoT Plug and Play Sample
 
   This [sample](https://github.com/Azure/azure-sdk-for-c/blob/main/sdk/samples/iot/paho_iot_hub_pnp_sample.c) connects an IoT Plug and Play enabled device (a thermostat) with the Digital Twin Model ID (DTMI) detailed [here](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/Thermostat.json). If a timeout occurs while waiting for a message from the Azure IoT Explorer, the sample will continue. If 3 timeouts occur consecutively, the sample will disconnect. X509 authentication is used.
 
-  To interact with this sample, **you must use the Azure IoT Explorer**.
-
-  <details><summary><i>How to interact with the Plug and Play sample:</i></summary>
-  <p>
-
-    The capabilities are listed below.
+  <details><summary><i>How to interact with the IoT Plug and Play sample:</i></summary>
 
-    <details><summary><b>Device Twin:</b></summary>
-    <p>
-
-    Two device twin properties are supported in this sample:
-    - A desired property named `targetTemperature` with a `double` value for the desired temperature.
-    - A reported property named `maxTempSinceLastReboot` with a `double` value for the highest temperature reached since device boot.
-    <br>
+  The easiest way to interact with this sample from the service side is to use Azure IoT Explorer.  To use the sample:
 
```

Comment:
I am going to assume these README changes, removals, and `IoT Hub Plug and Play Sample` heading renames are all intentional and correct. I know there were changes in main independent of the feature branch, and merging this README correctly, without loss of content was very tricky. It might be worth doing a careful review :)

Reply:
I'm assuming they're intentional and correct too:).  That said though I'll do another fresh eyes review presently and if anything is broken I'll update *this PR* and re-run gate for it.

Reply:
Re-review for the win.  

The `IoT Hub Plug and Play` were needed as it brings us inline with branding.  But there was an extra white space (ugh) and a duplicated section not related to anything.

The risk of these confusing future merge/git actions means I want this in this PR so we enter main clean.  

Otherwise good and i'll be pushing.

Reply:
Sounds good. As long as we aren't accidentally losing some of the content (that will be hard to trace back), we are good :)